CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ include_directories(third_party)
 
 set(lib_headers
     "${config_h}"
+    include/small/utils.h
```

Comment:
Nit: They usually call the header `util.h` (without 's' at the end).

Reply:
Done.